.buildkite/template.yml

```
@@ -1,12 +1,10 @@
-# template.yml
-# Use this as a starting point for your own Buildkite pipeline
-name: "Artifacts Pipeline Example"
-description: "An example repository you can use as a test project with Buildkite"
+name: "Bazel Python Package Example"
+description: "An example pipeline that builds, tests, and packages a Python library using Bazel."
+emoji: ":package:"
+languages:
+  - "Python"
+  - "Starlark"
+  - "YAML"
```

```suggestion
  - "Shell"
```

Not sure how that happened